config/routes.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  devise_for :users
```

Comment:
So if you will go with devise token auth they you will need to drop that, as it will add it's own version of that if I recall correctly